src/parsers/manifest/dash/js-parser/parse_from_document.ts

```diff
@@ -72,6 +72,10 @@ export default function parseFromDocument(
             if (loadedClock.length !== 1) {
               throw new Error("DASH parser: wrong number of loaded ressources.");
             }
+            if (typeof loadedClock[0].responseData !== "string") {
+              const newRet = initialRes.value.continue(null);
```

Comment:
Why changing the type (onError... | string) to another (null | string) here?
One level higher and we wouldn't really need this temporary `null` type which doesn't means much here.

What do you think?

Reply:
My point here still stands.

Also it may be nice to still emit the error as a `"warning"` when a retry happens.
Though I don't know if the manifest fetcher (in `src/core/fetchers/manifest`) doesn't already do it (to check).